lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
Maybe explicit is not the right term. _"number of milliseconds"_ does not apply to all timestamp types which is why I updated it to read _"timestamp"_ instead.

I would argue that developers who are using date validation are going to generally know what a timestamp is. If not, it is their responsibility to learn what a timestamp is…not the error messages.